warehouse/migrations/versions/c978a4eaa0f6_add_observation_related_name.py

```diff
@@ -60,17 +60,20 @@ def upgrade():
         ),
     )
 
+    # NOTE: Commenting out the following to avoid failing other migrations during tests.
```

Comment:
I don't think we've used ORM models in migrations very often - and specifically haven't used an ORM model that has later had a column changed.
I'm thinking that future data migrations ought to avoid using the live models.
This is also noted in https://stackoverflow.com/a/36087108/244037

```python
# Copy the model definitions into the migration script if
# you want the migration script to be robust against later
# changes to the models. ...
```